app.py

```
@@ -553,13 +865,118 @@ def index():
+            'interfaces': _interfaces_cache,
+            'lifx_interface': lifx_interface,
+            'sacn_interface': sacn_interface
+        })
+    
+    # Get interfaces in background thread to avoid blocking
+    def fetch_interfaces():
+        global _interfaces_cache, _interfaces_cache_time
+        try:
+            _interfaces_cache = get_network_interfaces()
+            _interfaces_cache_time = time.time()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)